templates/profile.php

```diff
@@ -75,7 +76,8 @@ function pmpromd_the_title($title, $post_id = NULL)
 				{
 					global $wpdb;
 					$user_nicename = $_REQUEST['pu'];
-					$user = $wpdb->get_row("SELECT * FROM $wpdb->users WHERE user_nicename = '" . esc_sql($user_nicename) . "' LIMIT 1");
+					$user_identifier = apply_filters( 'pmpromd_user_identifier', 'user_nicename' );

```

Comment:
This is used in multiple places, let's make a new function that has the hook in that and call that function here.